lib/engine/game/g_1825/game.rb

```diff
@@ -332,6 +265,9 @@ class Game < Game::Base
           },
         ].freeze
 
+        # MINORS = [
```

Comment:
Remove commented out code

Reply:
Was going back and forth how best to implement these ones. Done.